File: Selenium/Project/CRM_Activity4.py
# Import webdriver from selenium
from selenium import webdriver
from selenium.webdriver.common.by import By

# Define the login verification function
def verify_login(driver):
    
    # Navigate to the URL
    driver.get("https://alchemy.hguy.co/crm/")
    driver.maximize_window()

    # Log in to the application
    driver.find_element(By.ID, "user_name").send_keys("admin")
    driver.find_element(By.ID, "username_password").send_keys("pa$$w0rd")
    driver.find_element(By.ID, "bigbutton").click()

    # Verify the homepage by checking the dashboard text
    dashboard = driver.find_element(By.XPATH, "*//a[@id='tab0']").text
    assert dashboard == "SUITECRM DASHBOARD", "Dashboard text does not match!"
    print("End of method: Login Successful!")

# Main section to execute the function
if __name__ == "__main__":
    # Start the driver
    driver = webdriver.Chrome()

    try:
        # Call the login function
        verify_login(driver)
    finally:
        # Quit the browser
        driver.quit()

File: Selenium/Project/CRM_Activity4.java
package project;

import org.testng.annotations.Test;
import org.testng.annotations.BeforeClass;
import org.testng.annotations.Optional;
import org.testng.annotations.Parameters;
import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.chrome.ChromeDriver;
import org.openqa.selenium.firefox.FirefoxDriver;
import org.testng.Assert;
import org.testng.Reporter;
import org.testng.annotations.AfterClass;

public class CRM_Activity4 {
	
	//declare driver
		WebDriver driver;
		
		@BeforeClass
		  public void beforeClass() {
			driver = new ChromeDriver();
			  
		  }
			  
  @Test
  public void verifyLogin(WebDriver driver) {
	  
	//Launch application
	  Reporter.log("Launching application |");
	  driver.get("https://alchemy.hguy.co/crm/");
	  
	//login to application
	  driver.findElement(By.id("user_name")).sendKeys("admin");
	  driver.findElement(By.id("username_password")).sendKeys("pa$$w0rd");
	  driver.findElement(By.id("bigbutton")).click();
	  driver.manage().window().maximize();
	  
	  //verify homepage
	  String dashboard = driver.findElement(By.xpath("*//a[@id='tab0']")).getText();
	 Assert.assertEquals(dashboard, "SUITECRM DASHBOARD");
	  
	  
  }
  

  @AfterClass
  public void afterClass() {
	  
	  driver.quit();
  }

}
